data-transform.qmd

```diff
@@ -166,29 +165,32 @@ flights |>
   filter(month == 1 | 2)
 ```
 
-This works, in the sense that it doesn't throw an error, but it doesn't do what you want.
-We'll come back to what it does and why in @sec-boolean-operations.
+This "works", in the sense that it doesn't throw an error, but it doesn't do what you want because `|` first checks the condition `month == 1` and then checks the condition `2`, which is not a sensible condition to check.
+We'll learn more about what's happening here and why in @sec-boolean-operations.
 
 ### `arrange()`
 
 `arrange()` changes the order of the rows based on the value of the columns.
 It takes a data frame and a set of column names (or more complicated expressions) to order by.
 If you provide more than one column name, each additional column will be used to break ties in the values of preceding columns.
 For example, the following code sorts by the departure time, which is spread over four columns.
+We get the earliest years first, then within a year the earliest months, etc.
 
 ```{r}
 flights |> 
   arrange(year, month, day, dep_time)
 ```
 
-You can use `desc()` to re-order by a column in descending order.
-For example, this code shows the most delayed flights:
+You can use `desc()` to re-order the data frame based a column, in descending order.
```

Comment:
```suggestion
You can use `desc()` to re-order the data frame based a column in descending order.
```

But while the previous text was technically incorrect, I think it gets the message across more clearly.

Reply:
> You can use `desc()` to re-order the data frame by a column in descending order.

This seems like a good middle ground.